flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveTableSourceTest.java

```diff
@@ -199,6 +204,47 @@ public void testPartitionPrunning() throws Exception {
 		assertArrayEquals(new String[]{"2014,3,0", "2014,4,0"}, rowStrings);
 	}
 
+	@Test
+	public void testPartitionFilter() throws Exception {
```

Comment:
Could you also test the reading behavior in this case? By adding some data to the table and verify the results.

Reply:
Done